src/supermarket/config/locales/en.yml

```diff
@@ -124,7 +124,6 @@ en:
       only_cookbook_version: "You may not DELETE the only version of a cookbook. A cookbook must have at least one version."
       missing_cookbook_name: "Evaluation POST must contain cookbook_name."
       missing_cookbook_version: "Evaluation POST must contain cookbook_version."
-      missing_foodcritic_failure: "Evaluation POST must contain foodcritic_failure."
       missing_license_failure: "Evaluation POST must contain license_failure."
```

Comment:
Do we need to remove this as well?

Reply:
yea .. this message is not needed anywhere